apps/nextjs/src/env.ts

```diff
@@ -26,6 +28,10 @@ export const env = createEnv({
   runtimeEnv: {
     PORT: process.env.PORT,
     NODE_ENV: process.env.NODE_ENV,
+    ENABLE_DOCKER: process.env.NEXT_PUBLIC_ENABLE_DOCKER ? process.env.NEXT_PUBLIC_ENABLE_DOCKER === "true" : true,
+    ENABLE_KUBERNETES: process.env.NEXT_PUBLIC_ENABLE_KUBERNETES
+      ? process.env.NEXT_PUBLIC_ENABLE_KUBERNETES === "true"
```

Comment:
Don't use NEXT_PUBLIC env variables as they need to be defined during build time, which does not work for Homarr

Reply:
I may need some help with it. I'll ping u

Reply:
@Meierschlumpf I'm using these variables on the client side, so they need to be prefixed with NEXT_PUBLIC_ to work. However, since NEXT_PUBLIC_ variables are set at build time, this approach doesn't work for Homarr. I already tried placing them in shared and runtimeEnv, but that didn’t solve the issue.

What do you suggest for making these environment variables available on the client side while keeping them configurable at runtime?

Reply:
Depends where you want them to be available. If you really want them to be passed to the client I would suggest to just pass it either as a variable to a client component or if there are many components between, use a context where you configure it